styles.css

```diff
@@ -0,0 +1,23 @@
+body {
+  justify-content: center;
+  text-align: center;
+}
+
+
+#image {
+  max-width: 70rem;
+  height: 30rem;
+}
+
+#main {
+  background-color: rgb(184, 183, 183);
+}
+
+#div {
+  background-color: white;
+}
+
+#body {
```

Comment:
The styling should be done more carefully. At the moment it looks incomplete.

Reply:
- I think it will be better to use polite suggestion here instead of an order.  You can go in the lines of  `I'd like to kindly suggest that the styling should be done more carefully. . . . .` 🤗